instrumentation/hibernate/hibernate-6.0/src/main/java/com/solarwinds/opentelemetry/instrumentation/hibernate/v6_0/DrsaInstrumentation.java

```diff
@@ -69,41 +64,15 @@ public static void startMethod(
         return;
       }
 
-      try {
-        // We're using reflection here because this seems to be the best place we can intercept the
-        // raw sql before the PreparedStatement is generated and be able to make the span a parent
-        // of the execution step.
-
-        Class<?> clazz = drsa.getClass();
-        Field finalSql = clazz.getDeclaredField("finalSql");
-        finalSql.setAccessible(true);
-
-        String sql = (String) finalSql.get(drsa);
-        finalSql.setAccessible(false);
-        Context parentContext = currentContext();
-        if (!InstrumenterSingleton.instrumenter().shouldStart(parentContext, sql)) {
-          return;
-        }
-
-        Field statementCreator = clazz.getDeclaredField("statementCreator");
-        statementCreator.setAccessible(true);
-        Function<String, PreparedStatement> function =
-            (Function<String, PreparedStatement>) statementCreator.get(drsa);
-
-        if (!(function instanceof FunctionWrapper)) {
-          // We wrap the `statementCreator` field with our own wrapper to inject the context. This
-          // is less risky compared to modifying `finalSql` field.
-          statementCreator.set(drsa, new FunctionWrapper(function));
-        }
-        statementCreator.setAccessible(false);
-
-        context = InstrumenterSingleton.instrumenter().start(parentContext, sql);
-        scope = context.makeCurrent();
-        swoSql = sql;
-
-      } catch (Throwable throwable) {
-        LoggerFactory.getLogger().error("[Hibernate - v6] Error injecting context", throwable);
+      String sql = "";
```

Comment:
You're just keeping this sql variable as a placeholder in case needed in the future?

Reply:
Nope, needed something to pass around as the request for the instrumenter.